tests/lib/northbound/test_oper_data.refout

```diff
@@ -112,6 +112,10 @@ test# show yang operational-data /frr-test-module:frr-test-module
           }
         }
       ]
+    },
+    "c1value": 21,
+    "c2cont": {
+      "c2value": 2868969987
     }

```

Comment:
It looks wrong that we simultaneously have values from two different cases of a choise in our result. I think it's a violation of the RFC https://datatracker.ietf.org/doc/html/rfc8342#section-5.3. It says about operational datastore:
```
   Only semantic constraints MAY be violated.  These are the YANG
   "when", "must", "mandatory", "unique", "min-elements", and
   "max-elements" statements; and the uniqueness of key values.

   Syntactic constraints MUST NOT be violated, including hierarchical
   organization, identifiers, and type-based constraints.  If a node in
   <operational> does not meet the syntactic constraints, then it
   MUST NOT be returned, and some other mechanism should be used to flag
   the error.
```
This seems like a hierarchical organization to me that should not be violated. Should we use `lyd_validate_all` with `LYD_VALIDATE_OPERATIONAL` option somewhere at the end to make sure we don't produce an incorrect operational tree?

Reply:
This is on purpose. We should trust the backend to get this right, or it's a bug in the backend. I don't see a reason to extra-police this at the cost of CPU time and complexity.

Reply:
Sounds fine. We can always add a validation later if we decide that it's necessary. Merging.

Reply:
I did think about the fact that the test case violated the standard, but then figured it was also OK since the testing framework here is too simple and it highlighted the fact that the backend could do this. I have more tests i did locally that we can work back in once we have some implemented models with real choice/case stements. :)